tests/src/Traits/RdfEntityTrait.php

```diff
@@ -14,26 +14,29 @@
    *
    * If multiple RDF entities have the same name the first one will be returned.
    *
-   * @param string $title
-   *   The RDF entity title.
+   * @param string $label
+   *   The RDF entity label.
    * @param string $type
-   *   The RDF entity type.
+   *   Optional RDF entity type.
    *
    * @return \Drupal\rdf_entity\Entity\Rdf
    *   The RDF entity.
    *
    * @throws \InvalidArgumentException
    *   Thrown when an RDF entity with the given name and type does not exist.
    */
-  protected function getRdfEntityByLabel($title, $type) {
+  protected function getRdfEntityByLabel($label, $type = NULL) {
```

Comment:
Oh, how did this happen? You commented that since rdf_entities have unique titles, we can provide a default value of null and if not provided, search all the rdf entities. 
So my comment was that this is not true as it is unique title per bundle.